ObjectiveGitFramework.xcodeproj/project.pbxproj

```diff
@@ -1696,6 +1697,7 @@
 					"-force_load",
 					External/libgit2.a,
 					/usr/local/lib/libssh2.a,
+					/opt/homebrew/lib/libssh2.a,
```

Comment:
What about trying something like `-lssh2, -L/usr/local/lib, -L/opt/homebrew/lib` in these `OTHER_LDFLAGS` declarations?

Reply:
Nothing was here successful. 
I asked at 
* https://developer.apple.com/forums/thread/700085 and 
* https://stackoverflow.com/questions/71041332/xcode-architecture-independent-ldflags-arm64-x86-apple-silicon-m1

Reply:
Well, I'm glad you got *something* to work.